doc/security/vulnerabilities.rst

```diff
@@ -1255,3 +1255,27 @@ This has been fixed in main for v3.2.0
 
 - `PR 48733 fix for main
   <https://github.com/zephyrproject-rtos/zephyr/pull/48733>`_
+
+CVE-2022-3806
+-------------
+
+DoS: Invalid Initialization in le_read_buffer_size_complete()
+
+- `Zephyr project bug tracker GHSA-w525-fm68-ppq3
```

Comment:
Do we want to say "project bug tracker", or "vulnerability tracker"?

Reply:
The whole document is using "project bug tracker". We can do a batch change later though.